.travis.yml

```diff
@@ -50,6 +50,7 @@ jobs:
       - make install-minio
       script:
       - make integration-test
+      - make helm-test
```

Comment:
Can we make this a new job? It might run in parallel instead of waiting for all the apps to be tested. Will also decrease the run time